traveler-ios-ui/TravelerKitUI/ViewControllers/BookingItemSearch/BookingItemSearchSortFilterViewController.swift

```diff
@@ -0,0 +1,189 @@
+        bookingItemSearchFilters.categories = searchQuery!.categories
+        bookingItemSearchFilters.city = searchQuery!.city
+        bookingItemSearchFilters.country = searchQuery!.country
+        bookingItemSearchFilters.sortOption = sortData.sortOption
+        bookingItemSearchFilters.sortOrder = sortData.sortOrder
+        searchQuery = searchQuery!.filterSearchWith(bookingItemSearchFilters)
+        delegate?.bookingItemSearchSortFilterController(self, didSubmit: searchQuery!)
+    }
+    
+    func resetData() {
+        currentSort = defaultSort
+        
+        if let minFacetsPrice = currentSearchFacets?.minPrice.valueInBaseCurrency, let maxFacetsPrice = currentSearchFacets?.maxPrice.valueInBaseCurrency, minFacetsPrice <= maxFacetsPrice {
+            currentRange = minFacetsPrice...maxFacetsPrice
+        } else {
+            currentRange = fullRange
+        }
+    }
+    
+    @IBAction func reset() {
+        resetData()
+        tableView.reloadData()
+    }
+    
+    public override func numberOfSections(in tableView: UITableView) -> Int {
+        return 2
+    }
+    
+    public override func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        switch section {
+        case 0:
+            return 1
+        case 1:
+            return 1
+        default:
+            fatalError("Unhandled section; this should not happen!")
+        }
+    }
+    
+    public override func tableView(_ tableView: UITableView, titleForHeaderInSection section: Int) -> String? {
+        switch section {
+        case 0:
+            return "Sort"
+        case 1:
+            return "Filters"
+        default:
+            fatalError("Unhandled section; this should not happen!")
+        }
+    }
+    
+    public override func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        let cell = tableView.dequeueReusableCell(withIdentifier: CellIdentifiers.bookingItemSearchSortFilter, for: indexPath) as! BookingItemSearchSortFilterCell
+        cell.selectionStyle = .none
+        
+        switch indexPath {
+        case Paths.sort:
+            cell.titleLabel.text = "Order by"
+            cell.optionLabel.text = currentSort!.data.title
+        case Paths.price:
+            cell.titleLabel.text = "Price"
+            cell.optionLabel.text = String(format: "$%.2f - $%.2f", currentRange!.lowerBound, currentRange!.upperBound)
+        default:
+            fatalError("Unhandled sort/filter option.")
+        }
+        
+        return cell
+    }
+    
+    public override func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
+        switch indexPath {
+        case Paths.sort:
+            performSegue(withIdentifier: "sortSegue", sender: self)
+        case Paths.price:
+            performSegue(withIdentifier: "filterPriceSegue", sender: self)
+        default:
+            fatalError("Unhandled sort/filter option.")
```

Comment:
See comment above on fatalErrors, we want to stay away from those as much as possible.